server/routes/tests/delete.todo.test.js

```diff
@@ -0,0 +1,58 @@
+const request = require("supertest");
+const app = require("../../app");
+
+const expectedAllTodo = [
+    0: {
```

Comment:
There seems to be syntax errors with how this array is set up. Can be fixed with the following :)

```suggestion
    {
```